video/out/opengl/context_android.c

```diff
@@ -138,10 +138,22 @@ static bool android_reconfig(struct ra_ctx *ctx)
 {
     void *tmp = talloc_new(NULL);
     struct android_opts *opts = mp_get_config_group(tmp, ctx->global, &android_conf);
+    int w = opts->w, h = opts->h;
+    struct priv *p = ctx->priv;
```

Comment:
this line should probably be at the very top of the function for consistency reasons or whatever